powerlevel9k.zsh-theme

```diff
@@ -1228,11 +1228,18 @@ prompt_nvm() {
 
   node_version=$(nvm_version current)
   [[ -z "${node_version}" || ${node_version} == "none" ]] && return
+  if [[ "$node_version" == "system" ]]; then
+    node_version="$($(nvm which system) --version) system"
+  fi
 
   nvm_default=$(nvm_version default)
   [[ "$node_version" =~ "$nvm_default" ]] && return
 
-  $1_prompt_segment "$0" "$2" "magenta" "black" "${node_version:1}" 'NODE_ICON'
+  if [[ "${node_version:0:1}" == "v" ]]; then
```

Comment:
This just looks at the returned version and tries to get rid of the `v` at the beginning, right? Wouldn't it be more clear to parse it with a regular expression?
```zsh
[[ "${node_version}" =~ "^v([0-9.]+)" ]] && node_version="${match[1]}"
```